Wheres-Waldo-Frontend/src/tests/LeaderBoard.test.jsx

```diff
@@ -0,0 +1,11 @@
+/**
+ * @jest-environment jsdom
+ */
+import LeaderBoard from "../components/LeaderBoard";
+import { render, fireEvent, screen } from "@testing-library/react";
+import { MemoryRouter } from "react-router-dom";
+import { test } from "vitest";
+
+test("LeaderBoard renders", () => {
+    render(<MemoryRouter><LeaderBoard /></MemoryRouter>)
```

Comment:
Same here, would `expect` some text to exist. I don't think this test can ever fail currently